auth/user_mgt.go

```diff
@@ -155,10 +164,34 @@ func (u *UserToCreate) set(key string, value interface{}) *UserToCreate {
 	return u
 }
 
+// Converts a client format second factor object to server format.
+func convertMultiFactorInfoToServerFormat(mfaInfo MultiFactorInfo) (multiFactorInfoResponse, error) {
+	var authFactorInfo multiFactorInfoResponse
+	if mfaInfo.EnrollmentTimestamp != 0 {
+		authFactorInfo.EnrolledAt = time.Unix(mfaInfo.EnrollmentTimestamp, 0).Format("2006-01-02T15:04:05Z07:00Z")
```

Comment:
`2006-01-02T15:04:05Z07:00Z` Do we need all 9 fractions here? Example in the messaging API: https://github.com/firebase/firebase-admin-go/blob/23a1f17959238d989add41d562602b8bf04e71d6/messaging/messaging.go#L51

Reply:
Nit: Could we also make the format a const?

Reply:
Will accommodate these changes in a separate commit. Okay to keep the granularity of the 9 fractions for this.